### ISSUE TITLE
Font contrast lower than in UI

### ISSUE THREAD

**Issue by siphiuel** [tomislav/osx-terminal.app-colors-solarized#11]
Hi Tomislav!

Thanks so much for your work! I've been using Solarized for GUI Vim for quite a time, and it's great to have it available in Terminal.

I do have a minor issue though - text seems to have lower contrast (or maybe it's got a lighter shade) when viewing bare shell output, as compared to Terminal Vim.

I don't have any custom contrast settings in Vim. Maybe i can tweak something in the .terminal file? 

Attaching screenshot of a Tmux split pane (Terminal above, Vim below). Colors are actually similar in Tmux vs bare Terminal.

Will be grateful for any directions! Thanks again for your effort!
![solarized2](https://cloud.githubusercontent.com/assets/1019573/6573653/c946af36-c725-11e4-9fcc-c9e01a5389d5.png)

**Comment by tomislav** [comment]
Hi,

It's just how Terminal works, it seems to adjusts colors for some reason even if you set the proper RGB values for each color. You can try iTerm if it bothers you.

**Comment by siphiuel** [comment]
I see, thanks. I've also noticed that if 'Display ANSI Colors' checkbox is unchecked in Terminal Preferences, then Vim font colors become similar to the rest of terminal font colors.

**Comment by siphiuel** [comment]
Hi again. I did post an issue on StackOverflow yesterday (http://stackoverflow.com/questions/28964151/solarized-theme-in-os-x-terminal-having-brighter-font-colors-compared-to-vim), then found a solution just a few mins ago. Well, not exactly a solution - just wanted to let you know. I've simply tried a different Solarized Terminal scheme (https://github.com/tamul/solarized-osx-terminal-colors), and the colors there are not washed out. Don't know what is the reason, but that solved the issue for me.

Thanks for looking into this.
